packages/search/tests/integration/components/navi-search-result/definition-test.js

```diff
@@ -0,0 +1,54 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, findAll } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { set } from '@ember/object';
+
+module('Integration | Component | definition', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    assert.expect(1);
+
+    await render(hbs`<NaviSearchResult::Definition />`);
+
+    assert.equal(this.element.textContent.trim(), '');
+  });
+
+  test('displays results', async function(assert) {
```

Comment:
We should have a test that has to access the `extended` property of the record for the description as well

Reply:
test added 🙌